src/app/lists/CommentBox.tsx

```diff
@@ -0,0 +1,117 @@
+"use client";
+
+import FreeFormTextInput from "@/components/DataInput/FreeFormTextInput";
+import supabase from "@/supabase";
+import { Button } from "@mui/material";
+import React, { useState } from "react";
+import styled from "styled-components";
+
+interface CommentProps {
+    originalComment: string;
+}
+
+const HeaderAndButtonContainer = styled.div`
+    display: flex;
+    align-items: center;
+    width: 100%;
+    min-width: 8rem;
+    flex-direction: column;
+    gap: 1rem;
+    justify-content: space-between;
+    @media (min-width: 800px) {
```

Comment:
Don't think that is needed, it is quite common to use the same number repeatedly in media queries. Also it's not like that number is going to be wanted to be changed.

Reply:
@ChloeYiuSoftwire the standard practice for this sort of thing is to have predefined screen breakpoints (see something like how TailwindCSS manages it) so that actually would make sense. On the other hand, I was never too sure about why they should be standardized as I've never really come across an example where that would've made my life easier.